docker/Dockerfile

```diff
@@ -10,7 +10,8 @@ RUN pip wheel -r /build/pending.api/requirements.txt
 
 FROM python:3.8-slim
 RUN apt update -q -y && apt install -y --no-install-recommends git && rm -rf /var/lib/apt/lists/*
```

Comment:
Hi @kannabhargav, could you please let me know why git is ignored in the installation? Thanks!